src/translations/de.json

```diff
@@ -3,15 +3,23 @@
 
     "language-changed": "Die Anzeigesprache hat sich geändert. Möchtest du die Seite neu laden um alle Übersetzungen zu aktualisieren? Andernfalls könnten manche Übersetzungen nicht automatisch aktualisiert werden.",
 
-    "back-to": "Zurück zu",
+    "back-to-app": "Zurück zu {appName}",
+    "back-to-accounts": "Zurück zu meinen Konten",
+    "back-to-wallet": "Zurück zur {Wallet}",
+    "back-to-miner": "Zurück zum {Nimiq Miner}",
+    "back-to-faucet": "Zurück zur {Nimiq Faucet}",
+    "back-to-donation": "Zurück zur Spende",
```

Comment:
@danimoh I believe this is not a good translation. This is the _Donation Button Maker app_, not a donation itself.

Reply:
The app itself is called `Nimiq Donation`, thus this is a translation close to the original `Back to Nimiq Donation`.
However, to resolve this, I the donation app now identifies itself as `Donation Button Creator` and I adapted the translations accordingly.